integration_test/run.sh

```diff
@@ -19,12 +19,12 @@ PID1=$!
 sleep 3
 
 BLOCKFILTERARG=""
-if bitcoind -version | grep -q "v0\.\(19\|2\)"; then
+if bitcoind -version | grep -q "v\\(19\|20\|21\|22\)"; then
```

Comment:
The readme says we only support 0.21.0 and this PR title says its for 0.21 so we probably shouldn't include 22 here. That means you could leave this change out completely, right?

Reply:
@tcharding I removed the 22 from the PR, but this was precisely what fixed the integration tests for Bitcoin Core 22. Should I keep this in will this be added in another PR?

Reply:
If you are testing against v22 then the title of this PR is likely wrong.